lte/gateway/docker/services/python/Dockerfile

```diff
@@ -18,68 +18,46 @@ ARG CPU_ARCH=x86_64
 ARG DEB_PORT=amd64
 ARG OS_DIST=ubuntu
 ARG OS_RELEASE=focal
-ARG EXTRA_REPO=https://linuxfoundation.jfrog.io/artifactory/magma-packages-test
 
 FROM $OS_DIST:$OS_RELEASE AS builder
 ARG CPU_ARCH
 ARG DEB_PORT
 ARG OS_DIST
 ARG OS_RELEASE
-ARG EXTRA_REPO
 
 ENV MAGMA_DEV_MODE 0
 ENV TZ=Europe/Paris
 ENV MAGMA_ROOT=/magma
-ENV PYTHON_BUILD=/build
 ENV PIP_CACHE_HOME="~/.pipcache"
-ENV SWAGGER_CODEGEN_DIR /var/tmp/codegen/modules/swagger-codegen-cli/target
-ENV SWAGGER_CODEGEN_JAR ${SWAGGER_CODEGEN_DIR}/swagger-codegen-cli.jar
-ARG CODEGEN_VERSION=2.2.3
 ARG DEBIAN_FRONTEND=noninteractive
 
 RUN apt-get update && apt-get install -y \
   docker.io \
   git \
   lsb-release \
   libsystemd-dev \
-  libprotobuf-dev \
   pkg-config \
   python3-dev \
-  python3-eventlet \
   python3-pip \
-  python3-protobuf \
-  python3-pystemd \
-  ruby \
-  ruby-dev \
   sudo \
-  openjdk-8-jdk \
-  openjdk-8-jre-headless \
-  virtualenv \
   wget \
   && rm -rf /var/lib/apt/lists/*
 
-RUN gem install fpm
-
-COPY ./third_party/build/bin/aioeventlet_build.sh $MAGMA_ROOT/third_party/build/bin/aioeventlet_build.sh
-COPY ./third_party/build/lib/util.sh $MAGMA_ROOT/third_party/build/lib/util.sh
-WORKDIR /var/tmp/
-RUN /magma/third_party/build/bin/aioeventlet_build.sh && \
-  dpkg -i python3-aioeventlet*
-
-RUN mkdir -p ${SWAGGER_CODEGEN_DIR}; \
-  wget --no-verbose https://repo1.maven.org/maven2/io/swagger/swagger-codegen-cli/${CODEGEN_VERSION}/swagger-codegen-cli-${CODEGEN_VERSION}.jar -O ${SWAGGER_CODEGEN_JAR}
+# Download Bazel
+RUN  wget -P /usr/sbin --progress=dot:giga https://github.com/bazelbuild/bazelisk/releases/download/v1.10.0/bazelisk-linux-"${DEB_PORT}" && \
+  chmod +x /usr/sbin/bazelisk-linux-"${DEB_PORT}" && \
+  ln -s /usr/sbin/bazelisk-linux-"${DEB_PORT}" /usr/sbin/bazel
 
-COPY \
-    ./lte/gateway/python/Makefile \
-    ./lte/gateway/python/defs.mk \
-    ./lte/gateway/python/setup.py \
-    $MAGMA_ROOT/lte/gateway/python/
+COPY ./lte/gateway/python/integ_tests/s1aptests/ovs $MAGMA_ROOT/lte/gateway/python/integ_tests/s1aptests/ovs
```

Comment:
I'm curious as to why we specifically are moving the `ovs` directory. Why is this necessary now when it wasn't before. And why aren't we just moving the entire `integ_test` directory just as we do with the `load_tests` directory.

Reply:
It is used by the `packet_ryu_cli` target, which is a script that uses the `rest_api` in the `ovs` folder.

I don't know exactly why the load tests are copied here, I think the argument is that they might be useful in production deployments.